src/lib/expression/evaluation/like_matcher.hpp

```diff
@@ -82,19 +82,23 @@ class LikeMatcher {
    *    LikeMatcher{"%hello%"}.resolve(false, [](const auto& matcher) {
    *        std::cout << matcher("He said hello!") << std::endl;
    *    }
+   *
```

Comment:
Wasn't the main idea to get rid of `FixedStringVector::dictionary`?

Reply:
Yes, did I overlook something? The function is gone.

Reply:
Yes, did I overlook something? The function is gone.

Reply:
Unless Github is lying to me, fixed_string_vector.[ch]pp are not even touched.

Reply:
Whooopsie. Now it's really gone.